### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by krkos**
These are some improvements for "make check".
First fixes a run with multiple jobs, which is somewhat faster.
The second one allows running inside a virtualenv. That is beneficial as currently it is not possible to pass the tests in-tree.

### CODE REVIEW

**Comment by cejkato2** [comment]
According to https://docs.fedoraproject.org/en-US/packaging-guidelines/?fbclid=IwAR27NNqahk4aQOHyQF_YlzKw0wWQQwfbEciN-FAV-TQXSkjDCa3LdPulXcc#_shebang_lines /usr/bin/env is not a prefered way for deployed scripts.
However, current rpmbuild can handle it and rewrite shebangs. I can't look it up but I think rpmbuild failed in the past on the `/usr/bin/env` but it's ok now.
The question is, whether we really want to allow interpreter from environment - @krkos ?

**Comment by krkos** [comment]
> According to https://docs.fedoraproject.org/en-US/packaging-guidelines/?fbclid=IwAR27NNqahk4aQOHyQF_YlzKw0wWQQwfbEciN-FAV-TQXSkjDCa3LdPulXcc#_shebang_lines /usr/bin/env is not a prefered way for deployed scripts.
> However, current rpmbuild can handle it and rewrite shebangs. I can't look it up but I think rpmbuild failed in the past on the `/usr/bin/env` but it's ok now.

Oh, I see. I am sorry, I did not know, I am not a Fedora user.

> The question is, whether we really want to allow interpreter from environment - @krkos ?

Based on information above, probably not. The philosophy in RH distros is such and I believe it is the primary development and deployment platform (although Travis runs on Ubuntu), so I would advise to follow the guidelines.
On the other hand, the tests currently do not pass for me in-tree. I have to at least compile a "test" version of pytrap (for reference: https://github.com/krkos/Nemea-Framework/commit/2805fb9a33b1cea19882ccabd253ade8e2b77f96 ) and to install it so that it does not conflict with system stuff, I use virtualenv.

To sum it up, I do not care much about this change, I can keep a local copy. The testing problem should be fixed in another way. Perhaps by inspecting the Travis configuration?
Reverting said patch.

**Comment by cejkato2** [comment]
I'm sorry this complicates the testing, thank you for understanding.
